### ISSUE TITLE
Multi-container

### ISSUE THREAD

**Issue by so0k** [yofreke/docker-sentora#2]
Hi, wouldn't it make more sense to create a docker-compose.yml file pointing to separate mysql/apache/dovecot/ftpd/... services?

**Comment by so0k** [comment]
right now you're dockerfile isn't actually doing the installation, you run the container, then commit it to an image...

you're not running a supervisor or other process to ensure the services keep running. 

You're not mounting any volumes for the persistent data (apache sites / mysql / mailboxes / ..)

Did you say you use this in production?

**Comment by yofreke** [comment]
Correct, because of the early sentora install scripts, it was much easier to just run the installation script inside of the built container and then commit it.

Not sure about supervisor - I think sentora maybe now includes something for that in its install scripts?

Persistent data is handled by the backup.py script and the start-sentora-container.sh script.  See the readme for how that works.

This started as a side project and never really got finished.  After talking with one of the dudes over at sentora it seems like they have another option for running this stuff: https://github.com/trojanspike/sentora-dockerized which may be more up to date.

I have been meaning to get back to this and clean it up some, if you're looking to make use of it perhaps we could chat sometime and get things in a more usable state
